src/Files.Core/Data/Enums/GridViewSizeKind.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Core.Data.Enums
+{
+	public enum GridViewSizeKind
+	{
+		Compact = 0,
+		Small = 1,
+		Medium = 2,
+		Large = 3,
+		ExtraLarge = 4,
+		Five = 5,
+		Six = 6,
+		Seven = 7,
+		Eight = 8,
+		Nine = 9,
+		Ten = 10,
+		Eleven = 11,
```

Comment:
Naming ok?

Reply:
Not really but I don't know what else to use.

Reply:
Right. I will add comments in another pr.